src/cow-react/common/dumb/CurrencyInputPanel/__fixtures__/CurrencyInputPanel.tsx

```diff
@@ -1,11 +1,11 @@
-import { CurrencyInputPanel, CurrencyInputPanelProps } from 'pages/NewSwap/pureComponents/CurrencyInputPanel'
+import { CurrencyInputPanel, CurrencyInputPanelProps } from 'cow-react/common/dumb/CurrencyInputPanel'
```

Comment:
Instead of `dumb` we could maybe use `ui` or just `components` and the `smart` could be `containers` but not a big deal